statistics/histogram.go

```diff
@@ -738,6 +751,13 @@ func (c *Column) String() string {
 	return c.Histogram.ToString(0)
 }
 
+// MemoryUsage returns the total memory usage of a Column in B.
```

Comment:
what's `B` ?

Reply:
I copy them from chunk.MemoryUsage.. and forget to change it.